frontend/src/features/contacts/contact/detail/ContactInfoSubForm.tsx

```diff
@@ -0,0 +1,75 @@
+import { ContactMethodTypes } from 'constants/contactMethodType';
+import { ContactInfoField } from 'features/contacts/interfaces';
+import { Dictionary } from 'interfaces/Dictionary';
+import { IContactEntity } from 'interfaces/IContact';
+import * as React from 'react';
+import { Col } from 'react-bootstrap';
+import { phoneFormatter } from 'utils/formUtils';
+
+import * as Styled from '../../styles';
+import { getContactInfo } from './utils';
+
+interface IContactInfoSubFormProps {
+  contactEntity: IContactEntity;
+}
+
+const phoneTypes: Dictionary<string> = {};
+phoneTypes[ContactMethodTypes.WorkMobile] = 'Mobile';
+phoneTypes[ContactMethodTypes.WorkPhone] = 'Work';
+phoneTypes[ContactMethodTypes.PersonalMobile] = 'Home';
+phoneTypes[ContactMethodTypes.Fax] = 'Fax';
+phoneTypes[ContactMethodTypes.PersonalPhone] = 'Landline';
+
+const emailTypes: Dictionary<string> = {};
+emailTypes[ContactMethodTypes.WorkEmail] = 'Work';
+emailTypes[ContactMethodTypes.PersonalEmail] = 'Personal';
+
+const ContactInfoSubForm: React.FunctionComponent<IContactInfoSubFormProps> = ({
+  contactEntity,
+}) => {
+  const phoneNumbers: ContactInfoField[] = getContactInfo(
+    phoneTypes,
+    contactEntity?.contactMethods,
+  );
+  const emails: ContactInfoField[] = getContactInfo(emailTypes, contactEntity?.contactMethods);
+
+  return (
+    <>
+      <Styled.H2Primary>Contact info</Styled.H2Primary>
+      <Styled.RowAligned className="pb-4">
+        <Col md="2">
+          <strong>Email:</strong>
+        </Col>
+        <Col md="10">
+          {emails.length === 0 && <span>N.A</span>}
+          {emails.map((field: ContactInfoField, index: number) => (
+            <Styled.RowAligned key={'person-email-' + index}>
```

Comment:
nit: given that this is now a "generic" contact info sub-form - shouldn't the keys here be something like "contact-email-*" ? Seems odd to have key "person-*" when showing orgs